installation/configure-maven.adoc

```diff
@@ -113,3 +255,118 @@ If your Maven repository contains outdated artifacts, you might encounter Maven
 
 To resolve these issues, delete the cached version of your local repository that is located in the  `~/.m2/repository` directory to force a download of the latest Maven artifacts.
 
+== Build and run your own {PRODUCT_SHORT} v{VERSION_910} Maven Repository image on OpenShift
```

Comment:
I'm unsure about this whole section.

While it would be nice to provide a solution for customers/users who can't or don't want to create a Service Account with elevated privileges, there is an issue:
We don't ship the bamoe-maven-repository-image source with all other BAMOE sources (it should be inside the `bamoe-9.1.0-container-sources.zip`, but I think we missed it), which means that the Dockerfile used to build it is not available to download. I've pasted it in the instructions and that should work, but I'm not sure if it's a good idea to do so.

@eduardocerqueira @LightGuard @porcelli

Reply:
Might it make more sense to add it in the Known issues?

Reply:
This whole section? I'm not sure, it would look out of place there... Let's add a link there that points to this section here, WDYT?